DigitalExcellence/dex-frontend
# Bugfix/411 update budget

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Execute the lighthouse check and include the budget from the command line using:
`lighthouse https://staging.dex.software/home --budget-path=dex-frontend\budget.json --view`
2. Open the webpage and look for the header named Budgets. 
3. Check whether or not the over budget mark is checked. (it shouldn't be)

## Link to issue

Closes: #411
